packages/core/admin/admin/src/components/FormInputs/Date.tsx

```diff
@@ -27,7 +28,7 @@ const DateInput = forwardRef<HTMLInputElement, InputProps>(
             field.onChange(name, date);
           }}
           onClear={() => field.onChange(name, undefined)}
-          value={value}
+          value={value ? convertLocalDateToUTCDate(value) : undefined}
```

Comment:
A small suggestion to use `??` operator
```suggestion
          value={value ?? convertLocalDateToUTCDate(value)}
```

Reply:
thats not what this code is doing, its checking theres a value before we pass it to the `convertLocalDatetoUTCDate` func, so i'll probably keep it the same :)